src/Styleguide/Artwork/ImageCarousel.tsx

```diff
@@ -0,0 +1,209 @@
+import { Responsive } from "../Utils/Responsive"
+
+/**
+ * The image should be placed 100px from the top of the document and leave 60px
+ * for the buttons below the image.
+ */
+const ButtonsContainerHeight = 60
+const ButtonsContainerWithPageIndicatorsHeight = ButtonsContainerHeight + 20
+const ImageContainerViewportMargin = 100 + ButtonsContainerHeight
+
+const Container = styled(Flex)`
+  /* background-color: #0fdb82; */
+`
+
+const ImageAreaContainer = styled(Flex)`
+  width: 100%;
+  min-height: 450px;
+  height: calc(100vh - ${ImageContainerViewportMargin}px);
+  line-height: 0; /* Don’t introduce visual margins */
+`
+
+const ImageContainer = styled(Flex)`
+  height: 100%;
+  width: 100%;
+`
+
+const Image = styled.img`
+  cursor: zoom-in;
+  max-height: 100%;
+  max-width: 100%;
+`
+
+// TODO: Should Icon have this styling by default?
+const Button = styled.a`
+  ${Icon /* sc-selector */} {
+    vertical-align: middle;
+  }
+`
+
+const NavigationButtonsContainer = styled(Flex)`
+  width: 40px;
+  /* background-color: blue; */
+`
+
+const ActionButtonsContainer = styled(Flex)`
+  /* background-color: #f1af1b; */
+`
+
+const PageIndicator = styled.span`
+  &::after {
+    content: "•";
+  }
+
+  color: ${({ isHighlighted }: { isHighlighted: boolean }) =>
+    isHighlighted ? "#000" : "#d8d8d8"};
+`
+
+const PageIndicators = styled(Flex)`
+  ${PageIndicator} + ${PageIndicator} {
+    margin-left: 5px;
+  }
+
+  /* background-color: red; */
+`
+
+interface ImageCarouselProps {
+  src: string[]
+}
+
+interface ImageCarouselState {
+  currentImage: number
+}
+
+export class ImageCarousel extends React.Component<
+  ImageCarouselProps,
+  ImageCarouselState
+> {
+  state = {
+    currentImage: 0,
+  }
+
+  changeCurrentImage(by: number) {
+    this.setState({
+      currentImage: (this.state.currentImage + by) % this.props.src.length,
+    })
+  }
+
+  renderImageContainer() {
+    return (
+      <ImageContainer
+        flexDirection="row"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <Image
+          src={this.props.src[this.state.currentImage]}
+          // tslint:disable-next-line:no-console
+          onClick={() => console.log("Zoom")}
+        />
+      </ImageContainer>
+    )
+  }
+
+  renderNavigationButton(
+    iconName: "chevron-left" | "chevron-right",
+    changeCurrentImageBy: number
+  ) {
+    return (
+      <Responsive>
+        {({ xs }) => {
+          if (xs) {
+            return null
+          } else {
+            return (
+              <NavigationButtonsContainer
+                flexDirection="column"
+                justifyContent="center"
+              >
+                <Button
+                  href="#"
+                  onClick={e => {
+                    e.preventDefault()
+                    this.changeCurrentImage(changeCurrentImageBy)
+                  }}
+                >
+                  <Icon name={iconName} color="black" />
```

Comment:
We now have an arrow element as it's used all over: https://github.com/artsy/reaction/blob/master/src/Styleguide/Elements/Arrow.tsx